client/app/main/school/school.reports.controller.js

```diff
@@ -252,6 +252,102 @@ function SchoolReportsCtrl($scope, $timeout, Auth, uiGridGroupingConstants, Stud
     });
   };
 
+  $scope.outreachesGridOptions = {
+    rowHeight: 27,
+    enableSorting: true,
+    enableGridMenu: true,
+    enableFiltering: true,
+    treeRowHeaderAlwaysVisible: false,
+    exporterPdfDefaultStyle: {fontSize: 9},
+    exporterPdfTableHeaderStyle: {fontSize: 10, bold: true, color: 'grey'},
+    exporterPdfHeader: {
+      text: 'Chronically Absent Students',
+      style: 'headerStyle'
+    },
+    exporterPdfOrientation: 'landscape',
+    exporterPdfPageSize: 'LETTER',
+    exporterPdfMaxGridWidth: 500,
+    exporterPdfFooter: function(currentPage, pageCount) {
+      return {
+        text: currentPage.toString() + ' of ' + pageCount.toString(),
+        style: 'footerStyle'
+      };
+    },
+    exporterPdfCustomFormatter: function(docDefinition) {
+      docDefinition.styles.headerStyle =
+      {fontSize: 22, bold: true, color: '#265E6D'};
+      docDefinition.styles.footerStyle = {fontSize: 10, bold: true};
+      return docDefinition;
+    }
+  };
+
+  $scope.outreachesGridOptions.columnDefs = [{
+    name: 'student.studentId',
+    displayName: 'Student Id',
+    minWidth: 150,
+    cellTemplate: '<div class="ui-grid-cell-contents">' +
+                  '<a href="/student/{{row.entity.student._id}}">' +
+                  '{{row.entity.student.studentId}}</a>' +
+                  '</div>'
+    }, {
+      name: 'student.firstName',
+      displayName: 'First Name',
+      minWidth: 150
+    }, {
+      name: 'student.lastName',
+      displayName: 'Last Name',
+      minWidth: 150
+  }, {
+      name: 'total',
+      displayName: 'Total',
+      minWidth: 80
```

Comment:
lets add the `treeAggregationType: uiGridGroupingConstants.aggregation.SUM` to all of these. It looks better than "0"